main/provider/index.ts

```diff
@@ -904,7 +904,7 @@ export class Provider extends EventEmitter {
           address,
           symbol,
           decimals,
-          logoURI: tokenData.image || tokenData.logoURI || ''
+          image: tokenData.image || tokenData.logoURI || ''
```

Comment:
something is inconsistent here with the types. does this object expect an image or a string?